src/hooks/useWindowDimensions.js

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect, useCallback } from "react";
+
+export default function useWindowDimensions() {
+  const getWindowDimensions = useCallback(() => {
+    const { innerWidth: width } = typeof window !== "undefined" ? window : { innerWidth: 1920 };
+    return {
+      width,
+    };
+  }, []);
+
+  const [windowDimensions, setWindowDimensions] = useState(
+    getWindowDimensions()
+  );
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(getWindowDimensions());
+    }
+
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, [getWindowDimensions]);
+
+  return windowDimensions;
```

Comment:
Now that we have this component, it would be a good idea to refactor the css so that we're selecting styles.<classname> based on width instead of using @media so that we could set breakpoints across the app in the .env file. Not necessary for now, but if we revisit this in the future, it will make maintaining breakpoints or setting a common practice between apps much easier. Or we could look into using scss or sass 🤷